cli/azd/pkg/output/colors.go

```diff
@@ -48,6 +49,21 @@ func WithBackticks(text string) string {
 }
 
 // WithHyperlink wraps text with the colored hyperlink format escape sequence.
-func WithHyperlink(url string, text string) string {
-	return WithLinkFormat(fmt.Sprintf("\033]8;;%s\007%s\033]8;;\007", url, text))
+// When url and displayName are specified the displayName is used as the link display name
+// When url is an empty string the url is used as the link and the display name
+func WithHyperlink(url string, displayName string) string {
```

Comment:
@ellismg Has this worked for you on Apple Terminal? I just noticed that this ends up rendering as pure non-clickable text for me. There is `TERM_PROGRAM=Apple_Terminal` if we need to special case this.

Reply:
Yes - this does seem to just render as unclickable text for me as well in Apple Terminal on Sonoma 14.1.1.  I do think we should detect if `TERM_PROGRAM=Apple_Terminal` and render the links in some other form.

Reply:
If we do this, we'd have to worry a little about the text around the link working well with both versions.

For example:
"The Azure Portal" makes sense, but "The https://portal.azure.com" less so.